docsrc/index.rst

```diff
@@ -110,6 +110,8 @@ Tutorials
    tutorials/_rendered_examples/dynamo/torch_compile_resnet_example
    tutorials/_rendered_examples/dynamo/torch_compile_transformers_example
    tutorials/_rendered_examples/dynamo/torch_compile_advanced_usage
+   tutorials/_rendered_examples/dynamo/torch_compile_stable_diffusion
+   tutorials/_rendered_examples/dynamo/torch_compile_stable_diffusion
```

Comment:
Does this need to be listed twice?

Reply:
No, it should not have been - thanks for the catch, it is now fixed.